.travis.yml

```diff
@@ -11,20 +11,16 @@ matrix:
     # Barebone build to check tests pass when most of the tests must be skipped.
     # We need to use TEST=none to remove the test-data submodule.
     - env: INSTALL_TYPE=develop TEST=none NUMPYVER=1.11 TRAVIS_PYTHON_VERSION="2.7"
-    # Full build (including ds9 and xspec), Python 2.7, setup.py develop, astropy, matplotlib 1.5
-    - env: XSPECVER="12.9.1" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER=1.5 TRAVIS_PYTHON_VERSION="2.7"
-      sudo: required
-      dist: trusty
-    # As above, Python 3.5
+    # Full build (including ds9 and xspec), Python 3.5, setup.py develop, astropy, matplotlib 1.5
     - env: XSPECVER="12.9.1" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER=1.5 TRAVIS_PYTHON_VERSION="3.5"
       sudo: required
       dist: trusty
-    # As above, Python 3.6, setup.py install
-    - env: XSPECVER="12.9.1" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=1.5 TRAVIS_PYTHON_VERSION="3.6"
+    # As above, Python 3.6, setup.py install, xspec 12.10
+    - env: XSPECVER="12.10.0e" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2.0 TRAVIS_PYTHON_VERSION="3.6"
       sudo: required
       dist: trusty
-    # As above, xspec 12.9.1n
-    - env: XSPECVER="12.9.1" NUMPYVER="1.14" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2.0 TRAVIS_PYTHON_VERSION="3.6"
+    # As above, python 3.7, numpy 1.14, matplotlib 2.0
```

Comment:
The comment should read "matplotlib 3.0" I think?

Reply:
I think this is still outstanding, but I don't think it's an important enough change to force delaying merging this PR (given we want this in for #589) - as it is only a change in the comment and the comment can be fixed in another PR (which we're going to have to do when we add XSPEC 12.10.1).